sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubConnectionOptions.cs

```diff
@@ -25,6 +29,32 @@ public class EventHubConnectionOptions
         ///
         public EventHubsTransportType TransportType { get; set; } = EventHubsTransportType.AmqpTcp;
 
+        /// <summary>
+        ///   The amount of time to allow a connection to have no observed traffic before considering
+        ///   it idle and eligible to close.
+        /// </summary>
+        ///
+        /// <value>The default idle timeout is 60 seconds.  The timeout must be a positive value.</value>
+        ///
+        /// <remarks>
+        ///   If a connection is closed due to being idle, the Event Hubs clients will automatically
+        ///   reopen the connection when it is needed for a network operation.  An idle connection
+        ///   being closed does not cause client errors or interfere with normal operation.
+        ///
+        ///   It is recommended to use the default value unless your application has special needs and
```

Comment:
Does this line break need a <para tag, or will it be automatically preserved by the docs processing?

Reply:
Some really awesome human fixed the doc transform tools so that a blank line between two lines of text will automatically break into paragraphs.  😄